app/lib/three_scale/reform.rb

```diff
@@ -24,17 +24,14 @@ def save!(*)
     protected
 
     def merge_errors
-      definitions = property_definitions.values
-      definitions.each do |definition|
+      schema.each do |value|
```

Comment:
I found this change pretty tricky, because I don't really understand the documentation of the `reform` gem very well (not sure if it's mine, or the documentation's fault).

This seems to do the trick, although:

1. I am not sure if the behavior is 100% equivalent
2. The code says `The #errors method will be removed in Reform 3.0 core.
    def errors(*args)` https://github.com/trailblazer/reform/blob/v2.6.2/lib/reform/contract/validate.rb#L14
    
Also, there are just three forms, it seems, that use `reform`,and honestly, I think it might be possible to get rid of it at some point and simplify the implementation.

I think for the current purposes this rework should be enough, but probably we can refactor/remove this later.